db/schema.rb

```diff
@@ -10,7 +10,19 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_05_31_061410) do
+ActiveRecord::Schema.define(version: 2019_06_04_172236) do
+
+  create_table "announcements", force: :cascade do |t|
```

Comment:
That's weird. Did this not save from the announcements PR? Or does this need a rebase onto master?

Reply:
Doesn't it make sense that it's yesterday's date since that's when I last ran `rake db:migration`?